Week-2/Session6-Advanced CSS/Tic Tac Toe/tic-tac-toe.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML>
+<html>
+   <head>
+      <title>Tic-Tac-Toe</title> 
+      <link rel="stylesheet" href="styles.css">
+   </head>
+   <body>
+      <h1>Tic-Tac-Toe</h1>
+        
+      <div id="board">
+         <div class="x">X</div>
```

Comment:
please follow BEM naming conventions, throughout the project https://getbem.com/naming/